### ISSUE TITLE
Add travis ci to write software tests

### ISSUE THREAD

**Issue by danieldaeschle** [Jaffa/amazon-music#4]
Maybe after writing some tests you can add travis ci? :)

**Comment by Jaffa** [comment]
Yep. Big fan of automated testing; is there an introduction to unit testing in Python you can recommend?

**Comment by danieldaeschle** [comment]
I don't know an introduction at the moment.
But my default preferences are:

Install pytest for run tests with `pytest` command
Install pytest-sugar for nicer output. Use `pytest -p no:sguar` to run it with default output

### Directory called `tests`
which includes:

`conftest.py`
```python
import pytest

@pytest.fixture
def am():
    return AmazonMusic(...)
```

`my_test.py`
```Python
def test_sth(am):
    albums = am.list_albums()
    assert len(albums) > 0
```

Here is the [documentation](https://docs.pytest.org/en/latest/contents.html)

But to write tests, we have to insert credentials. This can be a little bit diffcult to solve...
